File: EntertainmentPortal/EP.Balda.Web/Models/Cell.cs
namespace EP.Balda.Models
{
    public class Cell
    {
        public char? Letter { get; set; }
        public int X { get; private set; }
        public int Y { get; private set; }

        public Cell(int x, int y)
        {
            X = x;
            Y = y;
        }
    }
}

File: EntertainmentPortal/EP.Balda/balda-client/src/app/game/models/cell.ts
export class Cell {
  constructor(public id: number, public mapId: number, public x: number, public y: number, public letter: string) {
    this.id = id;
    this.mapId = mapId;
    this.x = x;
    this.y = y;
    this.letter = letter;
  }
}
